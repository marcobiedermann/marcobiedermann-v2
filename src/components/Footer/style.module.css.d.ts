/* eslint-disable import/prefer-default-export */

export const footer: string;
