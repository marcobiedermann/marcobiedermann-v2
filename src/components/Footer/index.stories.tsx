/* eslint-disable import/no-extraneous-dependencies */

import { storiesOf } from '@storybook/react';
import React from 'react';
import Footer from '.';

storiesOf('Footer', module).add('default', () => <Footer>Content</Footer>);
