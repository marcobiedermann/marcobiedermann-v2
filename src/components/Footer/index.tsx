import classNames from 'classnames';
import React from 'react';
import styles from './style.module.css';

export interface FooterProps {
  className?: string;
}

const Footer: React.FC<FooterProps> = (props) => {
  const { className, ...otherProps } = props;

  return <footer className={classNames(className, styles.footer)} {...otherProps} />;
};

export default Footer;
