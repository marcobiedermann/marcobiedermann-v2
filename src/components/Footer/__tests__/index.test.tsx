/* eslint-disable import/no-extraneous-dependencies */

import { render } from '@testing-library/react';
import React from 'react';
import Footer from '..';

describe('Footer component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
