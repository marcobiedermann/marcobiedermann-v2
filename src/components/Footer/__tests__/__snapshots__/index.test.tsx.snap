// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<DocumentFragment>
  <footer
    class=""
  />
</DocumentFragment>
`;
